geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/AccumuloDataStoreParams.scala

```diff
@@ -9,23 +9,13 @@
 package org.locationtech.geomesa.accumulo.data
 
 
-import com.google.common.collect.ImmutableMap
-import org.apache.accumulo.core.client.Connector
-import org.geotools.data.DataAccessFactory.Param
-import org.geotools.data.Parameter
 import org.locationtech.geomesa.index.geotools.GeoMesaDataStoreFactory.GeoMesaDataStoreParams
 import org.locationtech.geomesa.security.SecurityParams
 import org.locationtech.geomesa.utils.geotools.GeoMesaParam
 
 // keep params in a separate object so we don't require accumulo classes on the build path to access it
 object AccumuloDataStoreParams extends GeoMesaDataStoreParams with SecurityParams {
 
-  val ConnectorParam =
```

Comment:
Is this us dropping support for passing in an existing connector instance?  (That's fine if it is, just checking.)

Reply:
right - we only used it in tests, and technically it's not serializable so it breaks the params contract... if someone feels strongly i don't mind putting it back in though.

Reply:
Right!  Not serializable; it is outta here!